lib/screens/feed_formula_page.dart

```diff
@@ -104,7 +104,7 @@ class _FeedFormulaPageState extends State<FeedFormulaPage> {
                   concentrateItems: concentrateItems,
                   cowRequirements: widget.cowRequirements,
                 ),
-                Image.asset('assets/sense-200px.png', height: 20),
+                // Image.asset('assets/sense-200px.png', height: 20),
```

Comment:
I will chat with Nikki about where we can put their logo instead.

Reply:
sounds good